lang/fr.json

```diff
@@ -33,15 +33,222 @@
+"Generate XORpad of the AGB (GBA) savegame partition.":"Générer les XORpads de la partition dédié aux sauvegardes de jeux AGB (GBA).",
+"Generate XORpad of the first firmware partition.":"Générer les XORpads de la première partition dédié au firmware.",
+"Generate XORpad of the second firmware partition.":"Générer les XORpads de la seconde partition dédié au firmwares.",
+"Generate XORpad of the CTR partition.":"Générer les XORpads de la partition CTR.",
+"Generate XORpad of the CTR (3DS) FAT 16 partition (CTR logical partition 0).":"Générer les XORpads de la partition FAT 16 CTR (3DS) (partition CTR logique 0)",
+"Dump":"Extraction",
+"Dumping options.":"Options d'extraction.",
+"Dump NAND image, separate partitions or files from the SysNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la SysNAND.",
+"Dump NAND image, separate partitions or files from the first EmuNAND.":"",
+"Dump NAND image, separate partitions or files from the second EmuNAND.":"",
+"Dump NAND image, separate partitions or files from the third EmuNAND.":"",
+"Image":"Образ",
+"Backup NAND raw image to file.":"",
+"Partitions":"",
+"Dump decrypted NAND partitions to file.":"",
+"Dump decrypted TWL (DSi) partition to file.":"",
+"Dump decrypted TWL (DSi) FAT16 partition (TWL logical partition 0) to file.":"",
+"Dump decrypted TWL (DSi) FAT12 photo partition (TWL logical partition 1) to file.":"",
+"Dump decrypted AGB (GBA) savegame partition to file.":"",
+"Dump decrypted first firmware partition to file.":"",
+"Dump decrypted second firmware partition to file.":"",
+"Dump decrypted CTR partition to file.":"",
+"Dump decrypted CTR (3DS) FAT 16 partition (CTR logical partition 0) to file.":"",
+"Files":"",
+"Dump files from the CTRNAND FAT16 partition.":"",
+"Inject":"",
+"Injection options.":"",
+"Inject NAND image, separate partitions or files to the SysNAND.":"",
+"Inject NAND image, separate partitions or files to the first EmuNAND.":"",
+"Inject NAND image, separate partitions or files to the second EmuNAND.":"",
+"Inject NAND image, separate partitions or files to the third EmuNAND.":"",
+"Restore NAND from raw image file.":"",
+"Inject unencrypted NAND partitions from file.":"",
+"Inject unencrypted TWL (DSi) partition from file.":"",
+"Inject unencrypted TWL (DSi) FAT16 partition (TWL logical partition 0) from file.":"",
+"Inject unencrypted TWL (DSi) FAT12 photo partition (TWL logical partition 1) from file.":"",
+"Inject unencrypted AGB (GBA) savegame partition from file.":"",
+"Inject unencrypted first firmware partition from file.":"",
+"Inject unencrypted second firmware partition from file.":"",
+"Inject unencrypted CTR partition from file.":"",
+"Inject unencrypted CTR (3DS) FAT 16 partition (CTR logical partition 0) from file.":"",
+"Inject files to the CTRNAND FAT16 partition.":"",
+"Copy":"",
+"Copy options.":"",
+"SysNAND to EmuNAND":"",
```

Comment:
You should not keep like that, there will be an empty string as a result when French choosen